meta-rz-common/include/core-image-renesas-sbc.inc

```diff
@@ -29,58 +29,22 @@ IMAGE_INSTALL_append = " \
     python3-matplotlib \
     python3-pip \
     apt-sources-config \
+    python3-smbus \
+    python3-smbus2 \
 "
 
-PACKAGECONFIG_append_pn-python3 = " tk"
-
 # Ignore vte-local-en-gb package because it has incompatible license GPL-3.0
 BAD_RECOMMENDATIONS += " vte-locale-en-gb"
 
-# Support wic image
-IMAGE_FSTYPES_append = " wic "
-
-# Remove unused rootfs types
-IMAGE_FSTYPES_remove = " tar.gz ext4 "
-
-# Linux bootloader flashing utility
-EXTRA_IMAGEDEPENDS += " bootloader-flasher-linux "
-
-# Windows bootloader flashing utility
-EXTRA_IMAGEDEPENDS += " bootloader-flasher-win "
-
-# Windows bootloader flashing readme
-EXTRA_IMAGEDEPENDS += " bootloader-flasher-readme "
-
-# Linux sd-creator-linux
-EXTRA_IMAGEDEPENDS += " sd-creator-linux "
-
-# Windows sd-creator-win
-EXTRA_IMAGEDEPENDS += " sd-creator-win "
-
-# sd-creator-readme
-EXTRA_IMAGEDEPENDS += " sd-creator-readme "
-
-# Linux uload-bootloader-linux
-EXTRA_IMAGEDEPENDS += " uload-bootloader-linux "
-
-# Windows uload-bootloader-win
-EXTRA_IMAGEDEPENDS += " uload-bootloader-win "
-
-# uload-bootloader-readme
-EXTRA_IMAGEDEPENDS += " uload-bootloader-readme "
-
-# uEnv.txt
-IMAGE_INSTALL_append = " uenv "
-
-# host-readme
-EXTRA_IMAGEDEPENDS += " host-readme "
+# Rootfs RZ SBC board
+IMAGE_INSTALL_append = " mmc-utils mtd-utils memtester minicom ca-certificates openssl dtc iputils ppp traceroute"
 
-# target-readme
-EXTRA_IMAGEDEPENDS += " target-readme "
+# Create folders for apt/dpkg
```

Comment:
This is a bug in Dunfell when I tried to install apt to RZ/G2L-SBC, so this function is implemented to address the issue.

```
# 1. When running APT commands like `apt-get update`, the following error occurs:
#    "E: The method driver /usr/lib/apt/methods/http could not be found."
#    This happens because APT requires the HTTP method driver located in 
#    `/usr/lib/apt/methods/`. However, in some cases, the driver files are installed 
#    in `/usr/lib64/apt/methods/` instead, causing APT to fail.
# 2. Additionally, the absence of the `status` and `available` files in `/var/lib/dpkg/` 
#    causes errors such as:
#    "E: Could not open file /var/lib/dpkg/status - open(2: No such file or directory)."
#    Creating the `status` and `available` files is necessary to resolve this issue.
```

@vudangRVC , please help me to add this comment to the PR, let me know if you need further adjustments.

Reply:
I updated in the new commit [c251fad](https://github.com/Renesas-SST/meta-renesas/commit/c251fad6229f3dd9e8a4f53440b591041c73ad42)